### PR TITLE
Add datastore and datastore cluster docs for vSphere

### PR DESCRIPTION
**Pull request by irozzo-1A**
* Align seed manifests
* Add datastore and datastore cluster documentation

### CODE REVIEW

File: content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md [hunk 1]
```diff
@@ -193,3 +193,40 @@ Upstream Kubernetes has been working on the issue for a long time now and tracki
...
+    datastore for dynamic volume provisioning.
+
+*Datastore Clusters* can only be used for the first purpose as it cannot be
+specified directly in [vSphere cloud configuration][vsphere-cloud-config].
+
+There are two places where Datastores and Datastore Clusters can be configured
+in KKP
+
+- At datacenter level (either with [Seed CRD]({{< ref "../../concepts/seeds" >}})
+    or [datacenters.yaml]({{< ref "../../concepts/seeds" >}})) is possible to
```
Comment (toschneck) [inline content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md:216]: Isn't the datacenter.yaml already depricated?

Comment (irozzo-1A) [reply content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md:216]: yes, it is but 2.15 still supports it and the majority of our customers is still using it

**Review by toschneck — COMMENTED** [review]
@irozzo-1A  Not sure if KKP change back to Kbuermatic was on purpose?
besides of this it looks good to me

File: content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md (hunk 1 above)
Comment (toschneck) [reply content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md:216]: ok, if its still supported  then we need to mention it

**Comment by irozzo-1A** [comment]
> @irozzo-1A Not sure if KKP change back to Kbuermatic was on purpose?
> besides of this it looks good to me

I just copied what gets generated in kubermatic repo: https://github.com/kubermatic/kubermatic/blob/master/docs/zz_generated.seed.yaml

File: content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md (hunk 1 above)
Comment (irozzo-1A) [reply content/kubermatic/master/requirements/cloud_provider/_vsphere.en.md:216]: BTW, the link for `datacenters.yaml` is wrong on this line. I will fix it

**Comment by toschneck** [comment]
Ok we shouldn't we automate this? I mean the chance is high that it doesn't get copied on changes

For the name changes, I think the new KKP names are the correct ones, so we may should change it in the source as well

**Comment by irozzo-1A** [comment]
> Ok we shouldn't we automate this? I mean the chance is high that it doesn't get copied on changes
Yes, definitely it's better to automate it or just put the link, cheaper solution. I don't think we should do it in the scope of this PR.
> For the name changes, I think the new KKP names are the correct ones, so we may should change it in the source as well
If we want KKP we should fix it on Kubermatic repo indeed.

**Comment by toschneck** [comment]
> Yes, definitely it's better to automate it or just put the link, cheaper solution. I don't think we should do it in the scope of this PR.

agree,  I just thought we already do it :sweat_smile: 
hmm maybe it's just broken I saw old commits from prow https://github.com/kubermatic/docs/commit/879633a4028913faa5a900633ea5c6e9f2fcd1b3

> If we want KKP we should fix it on Kubermatic repo indeed.

I think so, it was change by https://github.com/kubermatic/docs/pull/441 That's why I wouldn't overwrite it.

**Comment by youssefazrak** [comment]
@irozzo-1A lgtm minus the KKP being replaced by Kubermatic.

**Comment by irozzo-1A** [comment]
@toschneck @youssefazrak I think I address your point.

**Pull request merged**